File: 3-Bimestre/ex-10.cpp
#include <iostream>
using namespace std;

int main() {
    string senha;
    string senhaCorreta = "1234";
    int tentativas = 0;

    while (tentativas < 3) {
        cout << "Digite a senha: ";
        cin >> senha;

        if (senha == senhaCorreta) {
            cout << "Acesso permitido";
            break;
        } else {
            cout << "Senha incorreta" << endl;
            tentativas++;
        }
    }

    if (tentativas == 3)
        cout << "Acesso bloqueado";

    return 0;
}

File: 3-Bimestre/ex-10.py
senha_correta = '1234'
attempts = 0

while attempts < 3:
    senha = input('Digite a senha: ')
    if senha == senha_correta:
        print('Acesso permitido')
        break
    else:
        attempts += 1
        print('Senha incorreta')

if attempts == 3:
    print('Acesso bloqueado')
